worlds/hexcellsinfinite/locations.py

```diff
@@ -0,0 +1,24 @@
+from typing import TYPE_CHECKING
+from .types import LocData
+
+if TYPE_CHECKING:
+    from . import HexcellsInfiniteWorld
+
+def get_total_locations(world: "HexcellsInfiniteWorld") -> int:
+    return sum(1 for _ in location_table)
```

Comment:
```suggestion
    return len(location_table)
```
This function really does not seem like it needs to exist at all (probably can be said about a bunch of these functions that I think come from the skeleton), but if it does it should at least just be this I guess.